new-lamassu-admin/src/pages/Triggers/Triggers.js

```diff
@@ -11,10 +11,13 @@ import { Table as EditableTable } from 'src/components/editableTable'
 import { Switch } from 'src/components/inputs'
 import TitleSection from 'src/components/layout/TitleSection'
 import { P, Label2, H2 } from 'src/components/typography'
+import { ReactComponent as ReverseListingViewIcon } from 'src/styling/icons/circle buttons/listing-view/white.svg'
+import { ReactComponent as ListingViewIcon } from 'src/styling/icons/circle buttons/listing-view/zodiac.svg'
```

Comment:
According to the spec, the icons to use are the ones at `src/styling/icons/circle buttons/settings`, and not `listing-view`

Change the icon path and names here and lines 85 and 86

```suggestion
import { ReactComponent as ReverseSettingsIcon} from 'src/styling/icons/circle buttons/settings/white.svg'
import { ReactComponent as SettingsIcon } from 'src/styling/icons/circle buttons/settings/zodiac.svg'
```